core/src/cats/sql_get.cc

```diff
@@ -1344,10 +1342,11 @@ bool BareosDb::AccurateGetJobids(JobControlRecord* jcr,
   bstrutime(date, sizeof(date), StartTime + 1);
   jobids->clear();
 
+  char job_type = jr->JobType == JT_ARCHIVE ? 'A' : 'B';
```

Comment:
I added this for the following reason:
https://github.com/bareos/bareos/blob/be953a4c52559cdbf2dc231faa244d7643d0e3f7/core/src/dird/ua_restore.cc#L744-L764

When choosing option 12 in the restore menu, it tries to find appropriate incremental, differential and full restore jobs with `AccurateGetJobids`.  Without this change this will lead to a weird situation, where it will first tell you that a suitable restore was found just to then go ahead and say that there are actually none (since `AccurateGetJobids` just used to look at normal backups.)